flent/scripts/ss_iterate.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#Copyright (C) 2016  Matthias Tafelmeier
+
+#ss_iterate.sh is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+
+#ss_iterate.sh is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+
+#You should have received a copy of the GNU General Public License
+#along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+count=10
+interval=0.1
+host=localhost
+
+usage()
+{
+    echo "$0 -c count -I interval -H host -t target"
+}
+
+while getopts "c:I:H:t:p:" opt; do
+    case $opt in
+        c) count=$OPTARG ;;
+        I) interval=$OPTARG ;;
+        H) host=$OPTARG ;;
+        t) target=$OPTARG ;;
+    esac
+done
+
+if [ -z "$target" ]
+then
+    usage
+    exit 1
+fi
+
+command_string=$(cat <<EOF
+for i in \$(seq $count); do
+    ss -t -i -p -n state connected dst $target
```

Comment:
I think target is going to need quoting here as well, for the same reason as above.